invertedai/utils.py

```diff
@@ -33,6 +33,11 @@ def add_apikey(self, api_token: str = ""):
             raise error.InvalidAPIKeyError("Empty API key received.")
         self.session.auth = APITokenAuth(api_token)
 
+    def use_mock_api(self, use_mock: bool = True) -> None:
+        api_resources.mock_api = use_mock
+        if use_mock:
+            print('Using mock Inverted AI API - predictions will be trivial')  # TODO: replace with a suitable logger
```

Comment:
Do we have some uniform mechanism in place to issue warnings like this?

Reply:
iai.logger

Reply:
Is that safe in terms of circular imports?

Reply:
i think all the current reference of iai.logger happens inside the function so i never ran into any issue. We are already using it in this module

Reply:
OK. Did I use it correctly or should I do some extra formatting?

Reply:
looks good to me